Source/JavaScriptCore/runtime/IntlDurationFormat.cpp

```diff
@@ -359,13 +359,13 @@ static Vector<Element> collectElements(JSGlobalObject* globalObject, const IntlD
         double value = duration[unit];
 
         StringBuilder skeletonBuilder;
-        skeletonBuilder.append("rounding-mode-half-up"_s);
 
         switch (unit) {
         // 3.j. If unit is "seconds", "milliseconds", or "microseconds", then
         case TemporalUnit::Second:
         case TemporalUnit::Millisecond:
         case TemporalUnit::Microsecond: {
+            skeletonBuilder.append("rounding-mode-down"_s);
```

Comment:
Is this allowed in all ICU versions? Can you check when this is allowed, and ensure that it is enough old. (If not, you need to put runtime ICU version check).

Reply:
I did some research, but I couldn't determine exactly from which version this is supported. I will continue to investigate further, but we might end up implementing a version check.

However, `rounding-mode-down` is already used in [WebKit's `Intl.NumberFormat`](https://github.com/WebKit/WebKit/blob/bdc5e116475b8bd859d455fe8b37fd9d4cd07e42/Source/JavaScriptCore/runtime/IntlNumberFormatInlines.h#L198) without a runtime version check. Therefore, I believed it would work without any issues in the ICU versions that WebKit intends to support.

Reply:
The `rounding-mode-down` was introduced at the same time as the other `rounding-mode-*` by https://github.com/unicode-org/icu/commit/e27cf9ce39c98bedc661efce811985bb5926c0d8. The release containing this commit is ICU 62.1, which is the same for `rounding-mode-half-up`, so I think this change is fine as it is. What do you think?